docs/dev/pos/gui.rst

```diff
@@ -0,0 +1,37 @@
+JS access and inheritance
+=========================
+
+action_button
+-------------
+
+Here you will find explanation of how to get/inherit action_button POS objects.
+
+For example we have definition in /../../../*js file::
+
+    odoo.define('pos_reprint.pos_reprint', function (require) {
+    ...
+    screens.define_action_button({
+        'name': 'guests',
+        'widget': TableGuestsButton,
+        'condition': function()
+
+This defenition doesn't return class ReprintButton. So, we cannot inherit it in a usual way.
+
+In order to get access (inherit) to that object we need get instance of it using gui.
```

Comment:
In order to reach that object we need get instance of it using ``gui``. Then we can inherit it